src/daemon/krillserver.rs

```diff
@@ -221,7 +221,7 @@ impl KrillServer {
 
                     let repo_info: RepoInfo = pubserver.repo_info_for(&ta_handle)?;
 
-                    let ta_uri = format!("{}ta/ta.cer", uris.rrdp_base_uri());
+                    let ta_uri = format!("{}ta/ta.cer", service_uri);
```

Comment:
@timbru: Can you double check that this is okay here, perhaps it wasn't what you had in mind?

Reply:
I think this work: https://github.com/NLnetLabs/krill/pull/441

Is the right direction, but needs to be extended a bit.

Indeed base the default on 'service_uri' but allow overriding it. We could also go for env variables instead of config - perhaps this would be more inline with the 'KRILL_TESTBED_RSYNC' and 'KRILL_TESTBED_RRDP' settings.

Reply:
Or rather, I am also fine with just accepting this change here now. And fixing this in the other PR #441 that I just made.